docs/standard/library-guidelines/cross-platform-targeting.md

```diff
@@ -0,0 +1,99 @@
+
+**More Information**
+
+* [.NET Standard](https://docs.microsoft.com/en-us/dotnet/standard/net-standard)
+* [.NET API analyzer](https://docs.microsoft.com/en-us/dotnet/standard/analyzers/api-analyzer)
+
+## Multi-targeting
+
+Sometimes you need to access framework-specific APIs from your libraries. The best way to call framework-specific APIs is using multi-targeting, which builds your project for many frameworks rather than for just one.
+
+To shield your consumers from having to build for individual frameworks, you should strive to have a .NET Standard output plus one or more framework-specific outputs. With multi-targeting, all assemblies are packaged inside a single NuGet package. Consumers can then reference the same package and NuGet will pick the appropriate implementation. Your .NET Standard library serves as the fall-back library that is used everywhere, except for the cases where your NuGet package offers a framework-specific implementation. Multi-targeting allows you to use conditional compilation in your code and call framework-specific APIs.
+
+![NuGet package with multiple assemblies](./media/nuget-package-multiple-assemblies.png "NuGet package with multiple assemblies")
+
+**✔️ CONSIDER** targeting .NET implementations in addition to .NET Standard.
+
+> Targeting .NET implementations allows you to call platform-specific APIs that are outside of .NET Standard.
+>
+> Do not drop support for .NET Standard when you do this. Instead, throw from the implementation and offer capability APIs. This way, your library can be used anywhere and supports runtime light-up of features.
+
+**❌ DO NOT** use multi-targeting with .NET Standard if your source code is the same for all targets.
+
+> The .NET Standard assembly will automatically be used by NuGet. Targeting individual .NET implementations increases the `*.nupkg` size for no benefit.
+
+**✔️ CONSIDER** adding a target for `net461` when you're offering a `netstandard2.0` target. 
+
+> Using .NET Standard 2.0 from .NET Framework has some issues that were addressed in .NET Framework 4.7.2. You can improve the experience for developers that are still on .NET Framework 4.6.1 - 4.7.1 by offering them a binary that is built for .NET Framework 4.6.1.
+
+**✔️ DO** distribute your library using a NuGet package.
+
+> NuGet will select the best target for developer and shield them having to pick the appropriate implementation.
+
+**✔️ DO** use a project file's `TargetFrameworks` property when multi-targeting.
+
+```xml
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <!-- This project will output netstandard2.0 and net471 assemblies -->
+    <TargetFrameworks>netstandard2.0;net471</TargetFrameworks>
+  </PropertyGroup>
+</Project>
+```
+
+**✔️ CONSIDER** using [MSBuild.Sdk.Extras](https://github.com/onovotny/MSBuildSdkExtras) when multi-targeting for UWP and Xamarin as it greatly simplifies your project file.
+
+**More Information**
```

Comment:
So instead of having a "More info" section after each topic, you think it would be better to add "For more information, see XXX" throughout the discussion?

Reply:
In general, yes, that's better.  

We also have the "Related resources" at the end of many documents. In this PR, where you have one link in the More Information section, I'd put them inline. Where there is a list at the bottom of an article, I'd leave the "related resources" section.